Userland/Utilities/js.cpp

```diff
@@ -986,6 +988,11 @@ static bool parse_and_run(JS::Interpreter& interpreter, StringView source, Strin
         }
     };
 
+    if (!result.is_error())
+        last_value = JS::make_handle(result.value());
+    else
+        last_value = JS::make_handle(JS::js_undefined());
```

Comment:
This is undesirable: when using `_` in some expression and then you screw something up & it throws, the last value would be lost.

Since this feature is directly stolen from the Node.js REPL, I think we should do what they do:

```console
$ node
Welcome to Node.js v16.6.0.
Type ".help" for more information.
> "foo"
'foo'
> _
'foo'
> _ + x
Uncaught ReferenceError: x is not defined
> _
'foo'
> 
```

i.e.

```suggestion
```